.gitignore

```diff
@@ -1 +1,2 @@
 zig-cache/
```

Comment:
I assume this old one can be deleted?

Reply:
Ah yeah you're probably right. I'll fix that.